INSTRUCTION.md

```diff
@@ -0,0 +1,10 @@
+kubectl apply -f configMap.yml
+kubectl get configmap -n todoapp
+
+
+kubectl apply -f secret.yml
+kubectl get secret -n todoapp
+kubectl apply -f deployment.yml
+kubectl get deploymtnt -n todoapp
```

Comment:
There is a typo in the command `kubectl get deploymtnt`. It should be `kubectl get deployment` to correctly retrieve the deployment information.